locales/en.yml

```diff
@@ -75,7 +75,9 @@ en:
 
           Service:
               A service provided by sccli or systemd. For example:
-               docker, k8s, or openshift etc.
+               sccli service like docker, k8s, openshift etc
```

Comment:
We need to better scope this help as I think `sccli` service is not an easily understandable term.  I'll think on it